telemetry/README.md

```diff
@@ -0,0 +1,46 @@
+# Telemetry library
+
+This is a library for common telemetry functionality, especially subscribers for [Tokio tracing](https://github.com/tokio-rs/tracing)
+libraries.  The subscribers enable writing trace data to Jaeger, distributed tracing,
+common logs and metrics destinations, etc.
+
+Getting started is easy.  In your app:
+
+```rust
+  let config = telemetry::TelemetryConfig {
+    service_name: "my_app".into(),
+    ..Default::default()
+  };
+  telemetry::init(config);
+```
```

Comment:
Could we make this a regularly-compiled snippet? One approach would be a [code example](http://xion.io/post/code/rust-examples.html), which would log to stdout, and show the pretty colors when run.

Reply:
I'll add an example for sure.  I'd like to leave an example in the README though just for ease of copy and paste.... but I'm guessing the concern is code rot, in which case we can leave the snippet in the Rustic.